Makefile

```diff
@@ -13,6 +13,7 @@ gen:
 	go generate
 
 deps:
+	go get -u github.com/edsrzf/mmap-go
```

Comment:
We actually vendor project dependencies. Go supports automatic detection of dependent packages using the `vendor` directory. Helps checking in exact versions of dependencies instead of fetching stuff that may be broken. The current stuff was generated using govendor. Just do

```
go get -u github.com/kardianos/govendor
```

then, inside speed directory

```
govendor fetch github.com/edsrzf/mmap-go
```

This will modify `vendor.json` and add contents of the package.